pkg/resourcecreator/testdata/gcp_database_custom_env.yaml

```diff
@@ -90,5 +90,27 @@ tests:
                 secretKeyRef:
                   name: google-sql-myapplication
                   key: PWNED_PASSWORD
+  - apiVersion: sql.cnrm.cloud.google.com/v1beta1
+    kind: SQLUser
+    operation: CreateIfNotExists
+    match:
+      - type: subset
+        name: "sql user read"
+        resource:
+          metadata:
+            name: myapplication-read
+            namespace: mynamespace
+            annotations:
+              cnrm.cloud.google.com/deletion-policy: abandon
+              cnrm.cloud.google.com/project-id: team-project-id
+          spec:
+            instanceRef:
+              name: myapplication
+            host: ""
+            password:
+              valueFrom:
+                secretKeyRef:
+                    name: google-sql-myapplication
+                    key: PWNED_PASSWORD
```

Comment:
Same password for all users? I think they should have separate passwords.

Reply:
doh. rookie. fixing.